src/styles/variables.scss

```diff
@@ -0,0 +1,4 @@
+$main-accent: #060b35;
+$secound-accent: #00acdc;
+$third-accent: #616070;
+$font-family: "Roboto", sans-serif;
```

Comment:
[STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/17c41aa4-6308-422a-8db9-81b502efabc1)